### PR TITLE
Disable action button and add tooltip for HW controlled neurons in  Visibility Action

### PR DESCRIPTION
**Pull request by coskucinkilic**
# Motivation

Hardware wallet controlled neurons visibility cannot be changed for now, so we should disable the button and add tooltip to informing users.

# Changes

1. Use isNeuronControllableByUser instead of isNeuronControllable in NnsNeuronPublicVisibilityAction
2. Check if neuron is HW controlled and display Tooltip
3. Update NnsNeuronPublicVisibilityActionButton to have disabled property

# Tests

1. Add tests for NnsNeuronPublicVisibilityAction displaying disabled button and tooltip
2. Add tests for NnsNeuronPublicVisibilityActionButton is disabled when disabled prop is true

# Todos

- [ ] Add entry to changelog (if necessary).
not necessary

### CODE REVIEW

**Comment by dskloetd** [comment]
I think instead of hiding the button it should be disabled with a tooltip explaining why it's disabled.
I checked with Andrew and he agrees.

File: frontend/src/lib/components/neuron-detail/actions/NnsChangeNeuronVisibilityButton.svelte [hunk 1]
```diff
@@ -5,21 +5,25 @@
...
   let isPublic: boolean;
   $: isPublic = isPublicNeuron(neuron);
 
-  const openModal = () =>
-    openNnsNeuronModal({
-      type: "change-neuron-visibility",
-      data: { neuron },
-    });
+  const openModal = () => {
+    if (!disabled)
```
Comment (dskloetd) [inline frontend/src/lib/components/neuron-detail/actions/NnsChangeNeuronVisibilityButton.svelte:14]: Is this check necessary? Does the button generate click events even when it's disabled?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/components/neuron-detail/actions/NnsChangeNeuronVisibilityButton.svelte (hunk 1 above)
Comment (coskucinkilic) [reply frontend/src/lib/components/neuron-detail/actions/NnsChangeNeuronVisibilityButton.svelte:14]: It was causing tests to fail like the checkbox issue PageObjectClicks go through even the DOM elements are disabled.

Comment (coskucinkilic) [reply frontend/src/lib/components/neuron-detail/actions/NnsChangeNeuronVisibilityButton.svelte:14]: Should i merge this PR or do you have an idea to fix the underlying issue?

Comment (dskloetd) [reply frontend/src/lib/components/neuron-detail/actions/NnsChangeNeuronVisibilityButton.svelte:14]: Can you add a comment explaining this line and then merge?

**Pull request merged**